styles.css

```diff
@@ -116,48 +122,49 @@ h3 {
 /*=====================SECTION - ABOUT=======================*/
 
 section {
-    min-height: 390px;
-    background-color: #EAECEE;
+    background-color: #FFDD30;
     position: relative;
 }
 
 section div {
-    padding: 70px 0 0;
-    font-size: 1.5rem;
+    padding: 70px 70px 0;
```

Comment:
3 values in padding?

Reply:
yeah this is to say padding top left and right 70px and no padding on the bottom.